sqlglot/expressions.py

```diff
@@ -5331,6 +5331,12 @@ class Explode(Func):
     is_var_len_args = True
 
 
+# https://spark.apache.org/docs/latest/api/sql/#inline
+class Inline(Func):
+    arg_types = {"this": True, "expressions": False}
+    is_var_len_args = True
+
+
```

Comment:
According to the docs, `INLINE` accepts only a single expression i.e an array of structs; The `expressions` arg and `is_var_len_args = True` here are not needed in that case

Reply:
Tks! I will fix it